include/kernels/dslash_ndeg_twisted_clover_preconditioned.cuh

```diff
@@ -8,11 +8,11 @@
 namespace quda
 {
   
-  template <typename Float, int nColor, int nDim, QudaReconstructType reconstruct_, bool dynamic_clover_, bool asymmetric_>
+  template <typename Float, int nColor, int nDim, QudaReconstructType reconstruct_, bool asymmetric_>
```

Comment:
note that I've removed the `dynamic_clover` template argument since it's note required

Reply:
where is that set then?

Reply:
It's one of the compile-time properties that can be queried by constexpr functions, see `include/clover_field.h`:

```
445   /**
446      @brief Helper function that returns whether we have enabled
447      dyanmic clover inversion or not.
448    */
449   constexpr bool dynamic_clover_inverse()
450   {
451 #ifdef DYNAMIC_CLOVER
452     return true;
453 #else
454     return false;
455 #endif
456   }
```